test/cpp/run_tests.sh

```diff
@@ -9,6 +9,7 @@ BUILD_ONLY=0
 RMBUILD=1
 LOGFILE=/tmp/pytorch_cpp_test.log
 XLA_EXPERIMENTAL="nonzero:masked_select"
+export XLA_SYNC_BEFORE_ITEM_CALL=1
```

Comment:
If this is setting to the default(in other words it doesn't change any behavior) we should add a comment to make it clear.

Reply:
@ailzhang to clarify; where would the comment go? in run_tests.sh file or are you referring to somewhere else?